app/views/hyrax/embargoes/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Manage Embargoes</h1>
```

Comment:
If it's not much work, let's do it here in this PR.  Is there a good/related view style to copy/follow?